fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/WhereAdapter.java

```diff
@@ -42,20 +65,33 @@ public WhereAdapter where(String predicate) {
      * @return
      */
     public GroupByAdapter groupBy(String... expressions) {
-        select.addGroupBy(expressions);
-        return new GroupByAdapter(select);
+        GroupByClause gb = new GroupByClause();
+        select.setGroupByClause(gb);
+        return new GroupByAdapter(select, gb, expressions);
     }
-    
+
     /**
      * Start building the ORDER BY clause
      * @param expressions
      * @return
      */
     public OrderByAdapter orderBy(String... expressions) {
```

Comment:
WhereAdapter.orderBy(String[])" has 0 references

Reply:
The order by clause can be started from multiple places. In current usage it gets started group a GroupByAdapter, but as this is a framework, we provide other possibilities, based on the different types of SQL statement you can construct (not all sorted statements need a group by clause, etc. Statements may have a HAVING clause before the ORDER BY clause etc.